packages/tradershub/src/components/OptionsAndMultipliersSection/OptionsAndMultipliersHeading/OptionsAndMultipliersHeading.tsx

```diff
@@ -48,7 +47,9 @@ const OptionsAndMultipliersHeading = () => {
     return (
         <div className='flex flex-col items-start justify-between gap-16 lg:flex-row lg:gap-48'>
             <div className='gap-2 lg:flex lg:flex-col'>
-                {!isMobile && <Heading.H4>{title}</Heading.H4>}
+                <div className='md:hidden'>
+                    <Heading.H4>{title}</Heading.H4>
```

Comment:
are we not removing quill design?

Reply:
Yes @hamza-deriv 
On it right now in another PR https://app.clickup.com/t/20696747/TRAH-2878